# Generated by Django 4.0 on 2023-06-22 08:59

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('models_app', '0005_alter_subscription_authors'),
    ]

    operations = [
        migrations.AlterModelOptions(
            name='ingredientamount',
            options={'verbose_name': 'Количество ингридиентов'},
        ),
        migrations.AlterField(
            model_name='ingredientamount',
            name='ingredient',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='models_app.ingredient', verbose_name='Ингредиент'),
        ),
        migrations.AlterField(
            model_name='recipe',
            name='ingredients',
            field=models.ManyToManyField(to='models_app.IngredientAmount', verbose_name='Ингредиенты'),
        ),
        migrations.AlterField(
            model_name='subscription',
            name='authors',
            field=models.ManyToManyField(related_name='list_authors', to=settings.AUTH_USER_MODEL, verbose_name='Пользователи, на которых подписан'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer.', max_length=150, unique=True, verbose_name='Username'),
        ),
        migrations.AlterModelTable(
            name='shoppinglist',
            table='shopping_lists',
        ),
    ]
